src/components/Table/GridTable.stories.tsx

```diff
@@ -1849,3 +1854,146 @@ export function Headers() {
     </div>
   );
 }
+
+/**
+ * Shows how drag & drop reordering can be implemented with GridTable drag events
+ */
+export function DraggableRows() {
+  const nameColumn: GridColumn<OrderedRow<Data>> = {
+    header: "Name",
+    data: ({ name }) => ({ content: <div>{name}</div>, sortValue: name }),
+    spacer: " ",
+  };
+  const orderColumn: GridColumn<OrderedRow<Data>> = {
+    id: "order",
+    header: "Order",
+    data: (data, { row }) => row.order,
+    spacer: " ",
+  };
+  const actionColumn: GridColumn<OrderedRow<Data>> = {
+    header: "Action",
+    data: () => <div>Actions</div>,
+    spacer: " ",
+    clientSideSort: false,
+  };
+  const spacerRow: GridDataRow<OrderedRow<Data>> = {
```

Comment:
Thought: I think this spacer row is a good idea; afaict it like follows the mouse around to show where the being-dragged row will end up going?